### PR TITLE
LPS-61099

### PR DESCRIPTION
**Pull request by Ithildir**
@codyhoag, can you please review the changelog if the pull gets merged? Thank you!

### CODE REVIEW

**Comment by brianchandotcom** [comment]
ci:stop

**Comment by brianchandotcom** [comment]
@codyhoag this one is in upstream. Please review. Thx.

File: modules/sdk/gradle-plugins/CHANGELOG.markdown [hunk 1]
```diff
@@ -129,6 +129,19 @@ version 1.0.125.
...
+
+### Changed
+- [LPS-66853]: Update the [Liferay Source Formatter] dependency to version
+1.0.276.
+
+## 2.0.30 - 2016-09-09
+
+### Added
+- [LPS-61099]: Allow to overwrite the default value of
+`liferay.appServerParentDir` by setting the project property
```
Comment (codyhoag) [inline modules/sdk/gradle-plugins/CHANGELOG.markdown]: @Ithildir would you characterize `liferay.appServerParentDir` to be a property? Or something else?

Comment (Ithildir) [inline modules/sdk/gradle-plugins/CHANGELOG.markdown]: Yes, it's the `appServerParentDir` property of the `liferay` [extension object](https://docs.gradle.org/current/userguide/custom_plugins.html#sec:getting_input_from_the_build).

Comment (codyhoag) [inline modules/sdk/gradle-plugins/CHANGELOG.markdown]: Thanks, @Ithildir! :smiley:

**Pull request closed without merging**